pom.xml

```diff
@@ -34,6 +34,7 @@
         <module>hazelcast-spring</module>
         <module>hazelcast-build-utils</module>
         <module>hazelcast-all</module>
+        <module>hazelcast-karaf-itests</module>
```

Comment:
I would normally not introduce a new module but not sure if that is a must. The other option is to have this module in another repo but syncing hazelcast versions between repositories might be a hell of maintenance burden.

Reply:
It should be in a module which depends on `hazelcast-all`. Karaf will fetch the bundle from a maven repo, so it must have been `install`ed, not only `package`d, unfortunately. A different repo would help, but as you say has disadvantages as well.